ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -543,14 +558,21 @@ CMitsubaLoader::SContext::shape_ass_type CMitsubaLoader::loadBasicShape(SContext
 						auto meshbuffer = mesh->getMeshBuffer(i);
 						uint32_t offset = reinterpret_cast<uint8_t*>(it)-reinterpret_cast<uint8_t*>(newRGB->getPointer());
 						core::vectorSIMDf rgb;
-						for (uint32_t i=0u; meshbuffer->getAttribute(rgb, asset::EVAI_ATTR1, i); i++,it++) // should be upstreamed into the PLY loader
+						for (uint32_t i=0u; meshbuffer->getAttribute(rgb, 1u, i); i++,it++) // should be upstreamed into the PLY loader
```

Comment:
probably not, not many people expect their vertex colours to be decoded from SRGB encoding, and once you decode you either lose Alpha OR you need to expand to 64bits from 32bit.

Reply:
Could be a loader flag at some point @AnastaZIuk

Reply:
leaving as is for now